components/utils/time.js

```diff
@@ -1,4 +1,4 @@
-export default {
+const time = {
```

Comment:
I actually had to do this because on https://github.com/react-toolbox/react-toolbox/pull/366/files#diff-b92a8081cc5e0d62646a9cd36e0f58dcR181 I had to reference time